_search-plugins/sql/functions.md

```diff
@@ -18,166 +18,170 @@ The SQL plugin supports the following common functions shared across the SQL and
+| makedate           | `makedate(double, double) -> date`                                                     | `SELECT makedate(1945, 5.9)`                                                        |
+| maketime           | `maketime(integer, integer, integer) -> date`                                          | `SELECT maketime(1, 2, 3)`                                                          |
+| microsecond        | `microsecond(expr) -> integer`                                                         | `SELECT microsecond((time '01:02:03.123456'))`                                      |
+| minute             | `minute(expr) -> integer`                                                              | `SELECT minute((time '01:02:03'))`                                                  |
+| minute_of_day\*    | `minute_of_day(expr) -> integer`                                                       | `SELECT minute_of_day((time '01:02:03'))`                                           |
+| minute_of_hour\*   | `minute_of_hour(expr) -> integer`                                                      | `SELECT minute_of_hour((time '01:02:03'))`                                          |
+| month              | `month(date) -> integer`                                                               | `SELECT month(date('2020-08-26'))`                                                  |
+| month_of_year\*    | `month_of_year(date) -> integer`                                                       | `SELECT month_of_year(date('2020-08-26'))`                                          |
+| monthname          | `monthname(date) -> string`                                                            | `SELECT monthname(date('2020-08-26'))`                                              |
+| now                | `now() -> date`                                                                        | `SELECT now()`                                                                      |
+| period_add         | `period_add(integer, integer)`                                                         | `SELECT period_add(200801, 2)`                                                      |
+| period_diff        | `period_diff(integer, integer)`                                                        | `SELECT period_diff(200802, 200703)`                                                |
+| quarter            | `quarter(date) -> integer`                                                             | `SELECT quarter(date('2020-08-26'))`                                                |
+| second             | `second(time) -> integer`                                                              | `SELECT second((time '01:02:03'))`                                                  |
+| second_of_minute\* | `second_of_minute(time) -> integer`                                                    | `SELECT second_of_minute((time '01:02:03'))`                                        |
```

Comment:
```suggestion
| second_of_minute\* | `second_of_minute(time) -> integer`                                                    | `SELECT second_of_minute(time '01:02:03')`                                        |
```

Reply:
fixed in https://github.com/opensearch-project/documentation-website/pull/4232/commits/9442d3b1e8ec65a7c0e9b166073cf2b8281553d1